files/config.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Extreme Networks, Inc.
```

Comment:
Need to change it to the StackStom Authors. Check the `stackstorm/st2` codebase for example.

Reply:
I just copied existing file content, but I can update it.

Reply:
Yes, that would be great. It means we forgot to update those files. We however made changes to `st2` and `st2flow` codebases.

Reply:
Is that what you are referring to - https://github.com/StackStorm/st2flow/commit/2bde34a0bd0aa022150b81f710d8f4409d631180?

It seems like it's still missing "Copyright 2020 The StackStorm Authors." part. Or we don't want that anymore?

Reply:
It's like https://github.com/StackStorm/st2web/pull/759/files#diff-cb21033c4c4589a654b297f30aee73f6fac94040264a0acb09e816de1251d8e7R1